react/features/welcome/components/WelcomePage.web.js

```diff
@@ -66,6 +68,17 @@ class WelcomePage extends AbstractWelcomePage {
      * @returns {void}
      */
     componentDidMount() {
+        // FIXME: This is not the best place for this logic. Ideally we should
+        // use features/base/lib-jitsi-meet#initLib() action for this use case.
+        // But currently lib-jitsi-meet#initLib()'s logic works for mobile only
+        // (on web it ends up with infinite loop over initLib).
```

Comment:
I'm not reviewing this PR, I'm only mentioning concerns after hearing about this PR in today's morning standup:

1. If it's limited to Web only, it's fine with mobile. But when we're developing we should always account for our end-of-year goal to have Web and mobile merged as much as possible. I'll presume that's been accounted for here and the result is guided by higher priorities.

2. If it ever moves to mobile, please note that the WelcomePage is optional, we have a production use case in which the WelcomePage is disabled, and the WelcomePage only knows about the default domain (which is also configurable in Settings). As this PR currently only does Web-specific modifications, that's fine for now.